src/lib/components/workflow/dropdown-filter/workflow-datetime-filter.svelte

```diff
@@ -263,6 +268,13 @@
               </div>
             </div>
           {/if}
+          <MenuDivider />
+          <div class="flex w-full flex-row items-center gap-2 px-3 pt-2 m-1">
+            <Icon name="clock" aria-hidden="true" />
+            <span class="text-gray-500 font-normal text-xs"
+              >{translate('based-on-time-preface')} {localTime}
+            </span>
+          </div>
```

Comment:
Can we just use a disabled `<MenuItem />` here?

Reply:
I didn't use `MenuItem` since it has some default padding that we don't necessarily want 

<img width="452" alt="Screenshot 2023-08-22 at 3 52 38 PM" src="https://github.com/temporalio/ui/assets/15069288/be9c5c80-f06a-4160-8549-6dd47d66c7ad">

But I might just be able to use `!important` to override that 🤔

Reply:
Updated in [d3d0283](https://github.com/temporalio/ui/pull/1569/commits/d3d0283da54e78147261b11a78d51b5dc40dd263) ✔️